src/status_im/ui/screens/group/chat_settings/events.cljs

```diff
@@ -173,4 +172,4 @@
   :clear-history
   (fn [{{:keys [current-chat-id] :as db} :db} _]
     {:db (assoc-in db [:chats current-chat-id :messages] {})
-     ::chat-events/delete-messages current-chat-id}))
+     :delete-messages current-chat-id}))
```

Comment:
Any reason why you changed this to an unqualified keyword?

Reply:
Because implicitly qualified keywords are exceptionally bad idea when they are used outside the namespace where they were defined (and that was the case).
So it was `::delete-messages` keyword defined in `status-im.chat.handlers` ns, and `grep` for for `::delete-messages` revealed that the keyword is used only there - so far so good, but then I realised, then some other namespace `status-im.ui.screens.group.chat-settings.events` actually import the former namespace and refers to that keyword as `::chat-events/delete-messages`....

Reply:
Good catch! We should add this to guidelines somewhere, just to be aware if we see the same conundrum again